docs/events.md

```diff
@@ -68,3 +78,129 @@ point in time:
 ```js
 client.events.get({stream_position: '1408838928446360'}, callback);
 ```
+
+Enterprise Events
+======
+
+The Box API has an enterprise events endpoint that is available to admin users and service accounts.
+The SDK includes an `EnterpriseEventStream` class (which implements
+[stream.Readable](https://nodejs.org/api/stream.html#stream_readable_streams)) that automatically
+handles polling for events and delivering them to the application.
+
+* [Listening to the Enterprise Event Stream](#listening-to-the-enterprise-event-stream)
+* [Handling errors](#handling-errors)
+* [Get the Stream Position](#get-the-stream-position)
+
+Listening to the Enterprise Event Stream
+----------------------------------------
```

Comment:
These heading levels also need to be adjusted to be third-level headings, using the `### Title` notation